### PR TITLE
Add `data-e2e-link` attributes to 2fa options.

### PR DESCRIPTION
**Pull request by Tug**
We need attributes to grab 2fa links on the 2fa login page from wp-e2e-tests.
See Automattic/wp-e2e-tests#906

### Testing Instructions
- Boot the branch
- Go to the login page, inspect the page and check that the "Email me a login link" as a `data-e2e-link` attribute
- Login with a 2fa account with a valid number, check that both options (text and authenticator) have a `data-e2e-link` attribute.

### Reviews
- [x] Code
- [x] Product

### CODE REVIEW

**Comment by Tug** [comment]
Makes sense to use buttons here, since they don't behave like links and their action handlers may need to access some state (which we would lose if we would allow "open in a new tab" for instance).

File: client/login/wp-login/style.scss [hunk 1]
```diff
@@ -23,19 +23,23 @@ $image-height: 47px;
...
 		border-bottom: 1px solid lighten( $gray, 20% );
 		display: block;
 		font-weight: 500;
 		line-height: 4em;
 		padding: 0 24px;
+		text-align: left;
+		box-sizing: border-box;
+		width: 100%;
+		cursor: pointer;
 	}
```
Comment (stephanethomas) [inline client/login/wp-login/style.scss]: Super minor, but we could keep these rules ordered alphabetically.

**Comment by stephanethomas** [comment]
The code looks good to me.

**Comment by stephanethomas** [comment]
This works fine as well.

**Pull request merged**